apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressTokensByType.ts

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { UseFetchAddressProps } from '@/api/apiDataHooks/address/types'
+import useFetchAddressBalances from '@/api/apiDataHooks/address/useFetchAddressBalances'
+import useFetchSeparatedTokensByType from '@/api/apiDataHooks/utils/useFetchSeparatedTokensByType'
+
+interface UseFetchAddressTokensByType extends UseFetchAddressProps {
+  includeAlph: boolean
```

Comment:
I purposefully want this prop to be required and not optional so that the developer is forced to decide whether alph should be included in the results or not to avoid unnecessary computations when they are not needed. In the `useFetchAddressBalances` hook earlier in this diff, the `includeAlph` prop is optional and defaults to `true` because the hook is more general.